packages/sdk/src/react/hooks.ts

```
@@ -47,9 +57,399 @@ export function useModel<
+    const fetchData = useCallback(async () => {
+        // Wait until lock is released
+        while (isUpdating.current) {
+            await sleep(50);
+        }
+
+        // Lock function
+        isUpdating.current = true;
+
+        if (subscriptionRef.current) {
+            const [entities, clause] = await sdk.toriiQueryIntoHashedKeys(
+                fetchingRef.current!
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)